src/modules/UI.js

```diff
@@ -0,0 +1,84 @@
+import MyTodo from './todo.js';
+
+const UI = document.getElementById('todoList');
+
+class TodoList {
+  constructor() {
+    this.list = [];
+  }
+
+  addTodo = (description) => {
+    const todo = new MyTodo(description, false, this.list.length);
+    this.list.push(todo);
+    this.setLocalStorage(this.list);
+  }
+
+  deleteTodo = (index) => {
+    this.list = this.list.filter((todo) => todo.index !== index);
+    this.setLocalStorage(this.list);
+  }
```

Comment:
- [ ] So far the delete functionality is almost working well as intended ☺️ . However, when deleting there are little bugs we need to fix to ensure that the to-do list is professional 👍 .
     - I have noticed that when an item the index array length is not updated. Kindly ensure that the index of the items is updated to match the number of items in an array ☺️ .
     - In addition, when the item is deleted and you intend to delete the item after refreshing the webpage, all the tasks added are deleted. Kindly ensure that only the index of the task targeted is deleted 👌 .

Video demo 😎